### DISCUSSION TITLE
TuYa IH012-RT01 Disable Indicator LED

### DISCUSSION THREAD

**Discussion by vincentscode** [Koenkk/zigbee2mqtt#17441] [Devices]
Hi, I've recently connected a TuYa IH012-RT01. It works well, but whenever it detects motion an internal LED flashes blue, which is quite distracting.
Can this be disabled somehow? The current version does not seem to expose anything to disable the behavior.

**Comment by kapabel** [comment]
I am looking for the very same. Any luck with it?

**Comment by kapabel** [comment]
> Hi, I've recently connected a TuYa IH012-RT01. It works well, but whenever it detects motion an internal LED flashes blue, which is quite distracting.
> Can this be disabled somehow? The current version does not seem to expose anything to disable the behavior.

For information here is my solution: unscrew the back cover, take the battery out and place the device flat on the table. Then gently tap onto the PIR dome. You need to push it out the back. You may need more or less gentle force, start with low Rambo first. Then there is a round PCB in there. Take it out and unclip the dome. You will see the trouble LED. Use some electrical tape (someone told me dark nail polish etc. But I used tape) and cover this bad boy. At the end then put it all back together. Done!

**Reply by vincentscode** [reply]
That would be my backup solution, thanks for the detailed instructions.

**Comment by vincentscode** [comment]
I would have hoped something like this https://support.tuya.com/en/help/_detail/K9hw4fwh2z9dd can be utilized in software, but I do not know if such an endpoint exists for the specific device in question.
